submissions/develabr/oop/index.js

```diff
@@ -0,0 +1,76 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.args = ['species', 'name', 'gender', 'saying', 'friends'];

```

Comment:
Inhabitants are characterized rather by properties or features. No one says 'Name is my arg` or `Gender is my arg`